mula/scheduler/connectors/listeners/listeners.py

```diff
@@ -81,7 +82,10 @@ def callback(
         properties: pika.spec.BasicProperties,
         body: bytes,
     ) -> None:
-        self.logger.debug(" [x] Received %r", body)
+        """Callback function that is called when a message is received on the
+        queue.
+        """
+        self.logger.debug("Received message %r on queue %s", body, method.routing_key)
```

Comment:
(subjective) you might want to flip the order of arguments since it's probably easier to read the queue name before the (usually bigger) message body